### PR TITLE
updated CSP values in env.default

### LINKED ISSUE #1435: Update env.default to include CSP values
The https://github.com/mozilla/donate-wagtail/blob/master/env.default file currently does not come with preset CSP values similar ot the foundation's env.default, despite donate staging/prod having these values set.

We should ammend this file.

### PR DESCRIPTION
**Pull request by danielfmiranda**
Closes #1435 
Related PRs/issues #

## Checklist

**Changes in Models:**

- [x] [Are my changes backward-compatible]

### CODE REVIEW

File: env.default [hunk 1]
```diff
@@ -58,6 +58,15 @@ BASKET_SQS_QUEUE_URL=
 # See the Basket client docs https://basket-client.readthedocs.io/en/latest/install.html
 BASKET_API_ROOT_URL=https://basket-dev.allizom.org/
 
+# CSP config
+CSP_CONNECT_SRC=" * "
+CSP_FONT_SRC=" 'self' fonts.googleapis.com fonts.gstatic.com"
+CSP_FORM_ACTION=" 'self' https://webto.salesforce.com/ mozillasalescloud--mkl.cs4.my.salesforce.com "
```
Comment (Pomax) [inline env.default:64]: Do we need both of these for the code to run correctly on localhost?

File: env.default [hunk 2]
```diff
@@ -58,6 +58,15 @@ BASKET_SQS_QUEUE_URL=
...
+CSP_FRAME_SRC=" 'self' assets.braintreegateway.com c.paypal.com *.paypal.com https://www.google.com/recaptcha/api2/ https://tst.kaptcha.com/ "
```
Comment (Pomax) [inline env.default:65]: Do we need all of these for the code to run correctly on localhost? Also some of these don't have a protocol, let's make sure everything that is necessary to run locally also starts with https://

File: env.default [hunk 3]
```diff
@@ -58,6 +58,15 @@ BASKET_SQS_QUEUE_URL=
...
+CSP_IMG_SRC=" * data: "
```
Comment (Pomax) [inline env.default:66]: Do we use data: urls for images on donate? If so, then this makes sense, but do we?

File: env.default [hunk 4]
```diff
@@ -58,6 +58,15 @@ BASKET_SQS_QUEUE_URL=
...
+CSP_SCRIPT_SRC=" 'self' 'unsafe-inline' https://www.google-analytics.com/analytics.js  https://www.google-analytics.com/plugins/ua/ecommerce.js js.braintreegateway.com assets.braintreegateway.com www.paypalobjects.com c.paypal.com www.paypal.com https://www.google.com/recaptcha/api.js https://www.gstatic.com/recaptcha/releases/"
```
Comment (Pomax) [inline env.default:67]: Most of these look sensible, although we want some https:// prefixes for those bare domains.

File: env.default [hunk 5]
```diff
@@ -58,6 +58,15 @@ BASKET_SQS_QUEUE_URL=
...
 BASKET_API_ROOT_URL=https://basket-dev.allizom.org/
 
+# CSP config
+CSP_CONNECT_SRC=" * "
+CSP_FONT_SRC=" 'self' fonts.googleapis.com fonts.gstatic.com"
+CSP_FORM_ACTION=" 'self' https://webto.salesforce.com/ mozillasalescloud--mkl.cs4.my.salesforce.com "
+CSP_FRAME_SRC=" 'self' assets.braintreegateway.com c.paypal.com *.paypal.com https://www.google.com/recaptcha/api2/ https://tst.kaptcha.com/ "
+CSP_IMG_SRC=" * data: "
+CSP_SCRIPT_SRC=" 'self' 'unsafe-inline' https://www.google-analytics.com/analytics.js  https://www.google-analytics.com/plugins/ua/ecommerce.js js.braintreegateway.com assets.braintreegateway.com www.paypalobjects.com c.paypal.com www.paypal.com https://www.google.com/recaptcha/api.js https://www.gstatic.com/recaptcha/releases/" 
+CSP_STYLE_SRC=" 'self' 'unsafe-inline' fonts.googleapis.com fonts.gstatic.com "
```
Comment (Pomax) [inline env.default:68]: Let's add some https:// here, too

**Review by Pomax — CHANGES_REQUESTED** [review]

**Comment by danielfmiranda** [comment]
Hi Pomax, 
As discussed in our call earlier, I have updated the env.default file to match the variable names in default.py, and also added  a comment to let readers know that they can find the default values in the .py file. Please let me know if there are any more requested changes or questions. 
Thanks!

File: env.default [hunk 6]
```diff
@@ -58,6 +58,18 @@ BASKET_SQS_QUEUE_URL=
...
+# Default values for CSP items can be found in "donate/settings/defaults.py"
+CSP_DEFAULT_SRC=
+CSP_SCRIPT_SRC=
+CSP_IMG_SRC=
+CSP_FONT_SRC=
+CSP_STYLE_SRC=
+CSP_FRAME_SRC=
+CSP_CONNECT_SRC=
+CSP_BASE_URI=
+CSP_WORKER_SRC=
```
Comment (Pomax) [inline env.default:71]: we don't need these empty strings now. 
```suggestion
```

**Review by Pomax — CHANGES_REQUESTED** [review]

File: env.default (hunk 6 above)
Comment (danielfmiranda) [reply env.default]: Hi Pomax! Thanks for the heads up, I have removed them, and resubmitted for review 👍

**Review by Pomax — APPROVED** [review]

**Comment by Pomax** [comment]
Sorry for the run-around! Turns out that the fix was actually just to make sure the "where to look" instructions are in a place where folks look first.

**Pull request merged**